os-eap-sso/added/keycloak.sh

```diff
@@ -85,22 +75,22 @@ function configure_keycloak() {
     if [ -n "$SSO_SERVICE_URL" ]; then
       sso_service="$SSO_SERVICE_URL"
     fi
-  
+
+    if [ ! -n "${SSO_REALM:-master}" ]; then
```

Comment:
This will never be true.  Also, this either needs to be everywhere, or, after we test `-n "$SSO_REALM"`, we log the warning and `SSO_REALM=master` so the rest of the logic works correctly.

Reply:
@rcernich done, thanks.